src/js/components/productSupplier/create/InvalidItemsIndicator.jsx

```diff
@@ -2,31 +2,79 @@ import React from 'react';
 
 import PropTypes from 'prop-types';
 import { RiCheckboxCircleLine } from 'react-icons/all';
+import { RiErrorWarningLine } from 'react-icons/ri';
 
-import Translate from 'utils/Translate';
-
-const InvalidItemsIndicator = ({ className, errorsCounter }) => (
-  <div className={`invalid-items-indicator ${className}`}>
-    <span>
-      <RiCheckboxCircleLine className="mr-1" />
-      {errorsCounter}
-      {' '}
-      <Translate
-        id="react.productSupplier.form.invalidItemsIndicator.title"
-        defaultMessage="Item(s) require attention"
-      />
-    </span>
-  </div>
-);
+import Button from 'components/form-elements/Button';
+
+const invalidLines = {
```

Comment:
what's the purpose of those two consts? Could the name be more intuitive on what it does, not what it means?

Reply:
After getting to know the context, let's at least name it as `invalidLinesButton`/`validLinesButton`

Reply:
I was bothered about the naming here, so thanks for the suggestion.